wgpu-core/src/lib.rs

```diff
@@ -212,6 +214,8 @@ macro_rules! gfx_select {
             wgt::Backend::Dx12 => $global.$method::<$crate::backend::Dx12>( $($param),* ),
             #[cfg(windows)]
             wgt::Backend::Dx11 => $global.$method::<$crate::backend::Dx11>( $($param),* ),
+            #[cfg(unix)]
```

Comment:
why isn't this code using the cfg's established by the build.rs?

Reply:
Oh, I'm trying to remember for sure. I knew I should have added a comment when I ran into this. :man_facepalming:  I think there may have been an issue because it was in a macro. Like the CFG alias didn't work because it was evaluated before the alias existed somehow.

I may be wrong, though because I can't figure out the real reason that would happen looking back. I'll try to use the aliases and see if it still breaks or if I was confused earlier.

Reply:
Yeah, I tried it and it didn't work with the aliases. I'm not actually sure exactly why, but unfortunately in this macro at least we have to use the expanded cfg forms. I added a comment stating such.

:thinking: It's still a little confusing because I've used the cfg aliases in the other `macro_rules` macros I'm pretty sure.